Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -292,6 +292,70 @@ void CFrame::OpenGeneralConfiguration(wxWindowID tab_id)
   m_main_config_dialog->SetFocus();
 }
 
+void CFrame::CycleProfile(bool forward) {
+  //load the hotkey config
+  InputConfig* hk_conf = HotkeyManagerEmu::GetConfig();
+  auto hotkey_controller = hk_conf->GetController(0);
+
+  //load the wiimote config
+  InputConfig* wm_conf = Wiimote::GetConfig();
```

Comment:
As far as I can tell this can be a `const InputConfig*`

Reply:
Same as above, GetController function in InputConfig isn't and can't be const, so the InputConfig* can't be either.